src/commcare_cloud/ansible/roles/backups/templates/create_blobdb_backup.sh.j2

```diff
@@ -3,11 +3,13 @@ BACKUP_TYPE=$1
 MINUTES_TO_RETAIN_BACKUPS=$2
 DAYS_TO_RETAIN_BACKUPS=$2
 HOSTNAME=$(hostname)
+ENV="{{ env_name }}"
```

Comment:
@prem-fissionhq it looks like this resulted an an error for someone else trying to run this today. 

https://forum.dimagi.com/t/ansible-sudo-password-not-being-passed-to-deploy-commcare-hq-code/9015/2

I'm going to revert this PR and let you test it again to see if you can reproduce.